cardano-db-sync/src/Cardano/DbSync/Types.hs

```diff
@@ -198,16 +198,14 @@ data OffChainUrlType
   | OffChainVoteUrl !VoteUrl
   deriving (Eq, Generic)
 
-data OffChainHashType
-  = OffChainPoolHash PoolMetaHash
-  | OffChainVoteHash VoteMetaHash
-  deriving (Eq, Generic)
-
 instance Show OffChainUrlType where
-  show =
-    \case
-      OffChainPoolUrl url -> show url
-      OffChainVoteUrl url -> show url
+  show = getUrl
+
+getUrl :: OffChainUrlType -> String
+getUrl =
```

Comment:
could this be called `stringifyUrl` or `showUrl` just that the word "get" doesn't reflect that it's to do with converting a type to a string.